pages/attend/travel.md

```diff
@@ -37,6 +37,17 @@ US-RSE'24.
 
 - Hotel Andaluz Albuquerque, Curio Collection - 330 Tijeras Ave NW, Albuquerque, NM 87102 (0.1 miles away)
 
+## Travel Grants
+
+The US-RSE ’24 conference will again be offering travel grants via an application process.
+
+- Travel grants can be used for transportation, registration, and/or accommodations.
+- Meals will be provided at the conference on conference days.
+- Priority will be given to...
+  - those having an accepted submission at the US-RSE’24 conference.
```

Comment:
I have a nitpick: do we want "US-RSE’24" or "US-RSE ’24" (with/without the space)? Right now we have one of each.

Reply:
I vote no space. Other pages (as well as the header) have "US-RSE’24".

Reply:
no space is what we've used generally

Reply:
Based on consensus, I removed the space from the one "US-RSE" that had it.